### PR TITLE
new round of features for frontend

### LINKED ISSUE #53: Show machine orientation during production.

### PR DESCRIPTION
**Pull request by TarikViehmann**
This PR introduces some QoL features:
1. Only display welcome message when in spectator mode
2. Improve display of machine information in referee mode, including always showing their placement.
3. Swap Magenta and Cyan in referee mode. image
4. Add option to instruct machines 
image
5. Allow to customize the field via the frontend (in spectator mode using drag and drop 
https://github.com/robocup-logistics/rcll-refbox-frontend/assets/38430414/32b9d75b-ade1-4b81-a854-a54caf5cf22b

Fixes https://github.com/robocup-logistics/rcll-refbox-frontend/issues/53.

TODO: 
1. ~swap cyan and magenta around in spectator view~
2. ~lock machine positions to prevent accidents (require explicit checkmark before allowing to move them)~
3. merge https://github.com/robocup-logistics/rcll-refbox/pull/213

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctype html>
```
Comment (snoato) [inline index.html:1]: by convention doctype declarations are usually uppercase

File: src/components/shared/util/ConnectToDbBackend.vue [hunk 2]
```diff
@@ -77,7 +77,7 @@
                   (new Date(reportItem.end_time).getTime() -
                     new Date(reportItem.start_time).getTime()) /
                     1000,
-                  true
+                  true,
```
Comment (snoato) [inline src/components/shared/util/ConnectToDbBackend.vue:80]: u sure about this? or am I missing something

File: src/components/shared/util/ConnectToDbBackend.vue [hunk 3]
```diff
@@ -189,15 +189,15 @@ const filteredGameReportsList: ComputedRef<GameReport[]> = computed(() => {
     (report) =>
       report.report_name.includes(filter.value) ||
       report.teams[0].includes(filter.value) ||
-      report.teams[1].includes(filter.value)
+      report.teams[1].includes(filter.value),
```
Comment (snoato) [inline src/components/shared/util/ConnectToDbBackend.vue:192]: see above

File: src/components/spectator/GameInfoBoard.vue [hunk 4]
```diff
@@ -121,7 +121,7 @@
                 •
                 {{
                   formatTime(
-                    PRODUCTION_DURATION + OVERTIME_DURATION - game_time
+                    PRODUCTION_DURATION + OVERTIME_DURATION - game_time,
```
Comment (snoato) [inline src/components/spectator/GameInfoBoard.vue:124]: now I am getting confused

**Review by snoato — COMMENTED** [review]
Some questions

File: src/components/spectator/GameInfoBoard.vue (hunk 4 above)
Comment (TarikViehmann) [reply src/components/spectator/GameInfoBoard.vue:124]: that is automatic linting, they prefere to always have commas in the end.

File: index.html (hunk 1 above)
Comment (TarikViehmann) [reply index.html:1]: not according to the linter it seems :)

**Pull request merged**